raft/raftpb/raft.proto

```diff
@@ -109,7 +115,7 @@ enum ConfChangeTransition {
 message ConfState {
 	// The voters in the incoming config. (If the configuration is not joint,
 	// then the outgoing config is empty).
-	repeated uint64 voters = 1;
+	repeated uint64 voters = 1 ;
```

Comment:
Same question here, about the empty space.

Reply:
```suggestion
	repeated uint64 voters = 1;
```